packages/style-utilities/src/index.ts

```diff
@@ -1,9 +1,110 @@
-export * from './classNames/index';
-export * from './styles/index';
-export * from './utilities/index';
-export * from './interfaces/index';
-export * from './MergeStyles';
-export * from './cdn';
+export { AnimationClassNames, FontClassNames, ColorClassNames } from './classNames/index';
```

Comment:
This exports the same API. The change just addresses a lint warning to be specific about exports (i.e., no `export * ...`)